extensions/Worker.Extensions.EventGrid/src/EventGridOutputAttribute.cs

```diff
@@ -21,5 +21,10 @@ public EventGridOutputAttribute()
 
         /// <summary>Gets or sets the Topic Key setting. You can find information on getting the Key for a topic here: https://docs.microsoft.com/en-us/azure/event-grid/custom-event-quickstart#send-an-event-to-your-topic </summary>
         public string? TopicKeySetting { get; set; }
+
```

Comment:
Please also update the version and release notes. Are there any tests we can add to make sure this new Connection is working?